src/main/java/com/snowflake/core/util/HiveSyncTool.java

```diff
@@ -0,0 +1,248 @@
+/*
```

Comment:
if https://github.com/snowflakedb/snowflake-hive-metastore-connector/pull/23 is merged first, update the copyright header and package here.

Reply:
done